packages/builder/src/pages/builder/portal/settings/ai/index.svelte

```diff
@@ -159,74 +159,76 @@
   })
 </script>
 
-<Layout noPadding>
-  <Layout gap="XS" noPadding>
-    <div class="header">
-      <Heading size="M">AI</Heading>
-    </div>
-    <Body>
-      Connect an LLM to enable AI features. You can only enable one LLM at a
-      time.
-    </Body>
-  </Layout>
-  <Divider />
-
-  {#if !activeProvider && !$bannerStore}
-    <div class="banner">
-      <div class="banner-content">
-        <div class="banner-icon">
-          <img src={BBAI} alt="BB AI" width="24" height="24" />
-        </div>
-        <div>Try BB AI for free. 50,000 tokens included. No CC required.</div>
+{#if aiConfig}
+  <Layout noPadding>
```

Comment:
The flicker will go away, but will this not the full section to flash in?

Reply:
The UI won't load until the onMount lifecycle finishes?